src/odin/run.py

```diff
@@ -71,18 +66,7 @@ def start():
         schedule_afc_archive(schedule)
     if "spare" in config:
         schedule_spare_jobs(schedule, config["spare"])
+    if "data_dictionary" in config:
```

Comment:
scheduled data_dictionary job, will require update to devops repo to actually run.